wasm/src/main/scala/ir2wasm/WasmExpressionBuilder.scala

```diff
@@ -142,6 +143,14 @@ private class WasmExpressionBuilder private (
       case t: IRTrees.JSTypeOfGlobalRef    => genJSTypeOfGlobalRef(t)
       case t: IRTrees.JSLinkingInfo        => genJSLinkingInfo(t)
 
+      // array
+      case t: IRTrees.ArrayLength => genArrayLength(t)
+      case t: IRTrees.NewArray    => genNewArray(t)
+      case t: IRTrees.ArraySelect => genArraySelect(t)
+      case t: IRTrees.ArrayValue  => genArrayValue(t)
+      case t: IRTrees.Throw =>
+        instrs += UNREACHABLE // Implement Exception Handling (Throwable doesn't compile yet)
+        IRTypes.NothingType
```

Comment:
Undo the `case t: IRTrees.Throw =>` since #17 used `null.toString()` to achieve the same thing?

Or put it in `genThrow(t)` to keep the shape of this pattern match?